frontend/src/components/input/TextEditorMenu.vue

```diff
@@ -192,6 +198,65 @@ const buttons = computed(() => {
       isCollapsable: false,
       callback: () => props.editor.chain().focus().indent().run(),
     },
+    {
+      type: "insertTable",
+      icon: MdiTablePlus,
+      ariaLabel: "Tabelle einfügen",
+      group: "Tabelle",
+      isCollapsable: true,
+      isAlwaysCollapsed: true,
+      callback: () =>
+        props.editor
+          .chain()
+          .focus()
+          .insertTable({ rows: 3, cols: 3, withHeaderRow: false })
+          .run(),
+    },
```

Comment:
blocking: I understand that we do not want to test the library itself, however, we should test that our integration with the library works. Let's imagine the lib's new major version introduces a `commit()` method that needs to be called after `run()`. Without calling it, nothing happens. Renovate would just update the lib and the pipeline would be green.

I would argue for a minimal e2e test case like this:
* Uncollapse table menu -> check all buttons are displayed
* Click add table button and check that a table element is in editor (no details about table)